README.md

```diff
@@ -1 +1,224 @@
+
+```txt
+GET /v1/reports/{id}
+```
+
+#### Parameters
+
+| Name | Type   | In   | Description             |
+| ---- | ------ | ---- | ----------------------- |
+| `id` | string | path | The requested report ID |
+
+#### Response
+
+```txt
+Status: 200 OK
+```
+
+<details>
+  <summary>JSON response</summary>
+
+```json
+{
+  "benchmark": {
+    "records": [
+      {
+        "time": 152970821,
+        "code": 200,
+        "bytes": 46,
+        "error": "",
+        "events": [
+          {
+            "name": "GotFirstResponseByte",
+            "time": 2457696
+          }
+          // ...
+        ]
+      }
+      // ...
+    ],
+    "length": 10,
+    "success": 10,
+    "fail": 0,
+    "duration": 324953628
+  },
+  "metadata": {
+    "config": {
+      "request": {
+        "method": "GET",
+        "url": {
+          "scheme": "http",
+          "opaque": "",
+          "user": null,
+          "host": "echo.jsontest.com",
+          "path": "/title/ipsum/content/blah",
+          "rawPath": "",
+          "forceQuery": false,
+          "rawQuery": "",
+          "fragment": "",
+          "rawFragment": ""
+        },
+        "header": {},
+        "body": {
+          "type": "",
+          "content": ""
+        }
+      },
+      "runner": {
+        "requests": 10,
+        "concurrency": 10,
+        "interval": 50000000,
+        "requestTimeout": 2000000000,
+        "globalTimeout": 30000000000
+      }
+    },
+    "finishedAt": "2022-02-27T19:54:19.019717Z"
+  }
+}
+```
+
+</details>
+
+### List all the available statistics of previously reported run
+
+```txt
+GET /v1/stats
+```
+
+#### Parameters
+
+None. The list is always limited to the user's runs.
+
+#### Response
+
+```txt
+Status: 200 OK
+```
+
+<details>
+  <summary>JSON response</summary>
+
+```json
+[
+  {
+    "id": "lwkaFmtuCoeKSFhbndTC",
+    "tag": "my-custom-tag",
+    "finishedAt": "2022-02-27T19:54:19.019717Z"
+  },
+  {
+    "id": "XbqWESZWGC9iaXSWPDmu",
+    "tag": "my-other-custom-tag",
+    "finishedAt": "2022-03-03T18:00:19.019717Z"
+  }
+]
+```
+
+</details>
+
+### Get the statistics of a particular run
+
+```txt
+GET /v1/stats/{id}
+```
+
+#### Parameters
+
+| Name | Type   | In   | Description                                    |
+| ---- | ------ | ---- | ---------------------------------------------- |
+| `id` | string | path | The ID of the report whose stats are requested |
+
+#### Response
+
+```txt
+Status: 200 OK
+```
+
+<details>
+  <summary>JSON response</summary>
+
+```json
+{
+  "id": "lwkaFmtuCoeKSFhbndTC",
+  "tag": "my-custom-tag",
+  "finishedAt": "2022-02-27T19:54:19.019717Z",
+  "min": 152005288,
+  "max": 383330299,
+  "mean": 266585242,
+  "median": 266713182,
+  "standardDeviation": 51772085,
+  "deciles": [
+    152005288, 262461226, 263282513, 265239215, 265682968, 267743397, 268441975,
+    268631834, 269033707
+  ],
+  "code1xx": 0,
+  "code2xx": 1000,
+  "code3xx": 0,
+  "code4xx": 0,
+  "code5xx": 0
+}
```

Comment:
My bad, it is actually not exactly what is returned.
I have sent you some examples.